index.html

```diff
@@ -16,7 +16,7 @@ <h2 class="mb-xs-16">Educating and inspiring students by being <br class="hidden
 							Participate in Google Codein with <b>FOSSASIA</b> !
 						</p>
 						<a class="btn btn-lg btn-filled" href="https://codein.withgoogle.com/organizations/fossasia/">Take me Along !</a>
-						<a class="btn btn-lg" href="repos.html">Explore the site</a>
+						<a class="btn btn-lg" href="#mentors">Explore the site</a>
```

Comment:
Please revert

Reply:
Don't change anything that are not related to PR

Reply:
No. It is not done by me. It also came with merge. See more on - https://github.com/fossasia/gci17.fossasia.org/pull/359/files

Reply:
you deleted that's what I said.

Reply:
You means this line deleted by me - <a class="btn btn-lg" href="repos.html">Explore the site</a>

Reply:
Pls see - https://github.com/fossasia/gci17.fossasia.org/pull/359/files 
This delete and add from this PR. see it,